diff --git a/pyproject.toml.orig b/pyproject.toml
index d3084d5..d7990eb 100644
--- a/pyproject.toml.orig
+++ b/pyproject.toml
@@ -70,16 +70,13 @@ matplotlib = ">=3.6.0"
 pyyaml = ">=5.3"
 defusedxml = "^0.7.1"
 pillow = ">=9.4"
-requests = { version = ">=2.26.0,<=2.32.3", optional = true }
-tqdm = { version = ">=4.62.3,<=4.67.0", optional = true }
+requests = ">=2.26.0"
+tqdm = ">=4.62.3"
 # pandas: picked lowest major version that supports Python 3.8
-pandas = { version = ">=2.0.0", optional = true }
-pandas-stubs = { version = ">=2.0.0.230412", optional = true }
+pandas = ">=2.0.0"
+pandas-stubs = ">=2.0.0.230412"
 opencv-python = ">=4.5.5.64"
 
-[tool.poetry.extras]
-assets = ["requests", "tqdm"]
-metrics = ["pandas", "pandas-stubs"]
 
 [tool.poetry.group.dev.dependencies]
 twine = "^5.1.1"
@@ -110,15 +107,6 @@ mkdocs-jupyter = "^0.24.3"
 mkdocs-git-committers-plugin-2 = "^2.4.1"
 mkdocs-git-revision-date-localized-plugin = "^1.2.4"
 
-[tool.poetry.group.typecheck]
-optional = true
-
-[tool.poetry.group.typecheck.dependencies]
-types-pyyaml = "^6.0.12.20240808"
-types-cffi = "^1.16.0.20240331"
-types-requests = "^2.32.0.20240712"
-types-tqdm = "^4.66.0.20240417"
-
 
 [tool.bandit]
 target = ["test", "supervision"]
