diff --git a/pyproject.toml b/pyproject.toml.orig
index c9aa384..9308b25 100644
--- a/pyproject.toml
+++ b/pyproject.toml.orig
@@ -55,16 +55,12 @@ matplotlib = ">=3.6.0"
 pyyaml = ">=5.3"
 defusedxml = "^0.7.1"
 pillow = ">=9.4"
-requests = { version = ">=2.26.0,<=2.32.3", optional = true }
-tqdm = { version = ">=4.62.3,<=4.66.5", optional = true }
-# pandas: picked lowest major version that supports Python 3.8
-pandas = { version = ">=2.0.0", optional = true }
-pandas-stubs = { version = ">=2.0.0.230412", optional = true }
+requests = ">=2.26.0,<=2.32.3"
+tqdm = ">=4.62.3,<=4.66.5"
+pandas = ">=2.0.0"
+pandas-stubs = ">=2.0.0.230412"
 opencv-python = ">=4.5.5.64"
 
-[tool.poetry.extras]
-assets = ["requests", "tqdm"]
-metrics = ["pandas", "pandas-stubs"]
 
 [tool.poetry.group.dev.dependencies]
 twine = "^5.1.1"
