diff --git a/pyproject.toml b/pyproject.toml.orig
index fcb9128b..d16bc89e 100644
--- a/pyproject.toml
+++ b/pyproject.toml.orig
@@ -48,15 +48,11 @@ scipy = [
 matplotlib = ">=3.6.0"
 pyyaml = ">=5.3"
 defusedxml = "^0.7.1"
-opencv-python = { version = ">=4.5.5.64", optional = true }
-opencv-python-headless = ">=4.5.5.64"
-requests = { version = ">=2.26.0,<=2.32.3", optional = true }
-tqdm = { version = ">=4.62.3,<=4.66.4", optional = true }
+opencv-python = { version = ">=4.5.5.64"}
+requests = { version = ">=2.26.0,<=2.32.3"}
+tqdm = { version = ">=4.62.3,<=4.66.4"}
 pillow = ">=9.4"
 
-[tool.poetry.extras]
-desktop = ["opencv-python"]
-assets = ["requests", "tqdm"]
 
 [tool.poetry.group.dev.dependencies]
 twine = "^5.1.1"
