diff --git a/pyproject.toml.orig b/pyproject.toml
index ae252156..3fbec9c4 100644
--- a/pyproject.toml.orig
+++ b/pyproject.toml
@@ -85,8 +85,8 @@ tqdm = ">=4.62.3"
 # pandas: picked lowest major version that supports Python 3.8
 # pandas 2.2.3 has been released with support for Python 3.13
 pandas = [
-    { version = ">=2.0.0", python = "<3.13", optional = true },
-    { version = ">=2.2.3", python = ">=3.13", optional = true },
+    { version = ">=2.0.0", python = "<3.13"},
+    { version = ">=2.2.3", python = ">=3.13"},
 ]
 
 opencv-python = ">=4.5.5.64"
